hack/e2e_tests.sh

```diff
@@ -27,6 +27,7 @@ GINKGO_ARGS=${GINKGO_ARGS:-}
 ENABLE_NFD=${ENABLE_NFD:-true}
 ENABLE_GPU_OPERATOR=${ENABLE_GPU_OPERATOR:-true}
 ENABLE_LOCAL_PATH_PROVISIONER=${ENABLE_LOCAL_PATH_PROVISIONER:-true}
+E2E_TIMEOUT=${E2E_TIMEOUT:-"30"} # Int type, will be converted to minutes
```

Comment:
can we rename it to `E2E_TIMEOUT_SECONDS` and default to "1800" to make it explicit.

Reply:
Done